full-moon/src/ast/parsers.rs

```diff
@@ -1383,6 +1385,32 @@ cfg_if::cfg_if! {
                 ))
             }
         );
+
+        // Roblox Compound Assignment
+        #[derive(Clone, Debug, Default, PartialEq)]
+        struct ParseCompoundAssignment;
+        define_parser!(

```

Comment:
I'm almost certain this needs to be define_roblox_parser. Make sure you test both with and without the Roblox feature flag.

Reply:
I added this here because it is under the conditional statement (this is also where the type parsers were, which just use define_parser)